### ISSUE TITLE
Define EU conformance exemption metadata

### ISSUE THREAD

**Issue by mattgarrish** [w3c/publ-a11y#175]
@gregoriopellegrino raised the need in the EU to identify publications that fall under one of the three exemptions from the accessibility requirements. ONIX is adding three new values to code list 196 to address this (75-77).

We're currently not aware of any existing metadata property that could be used to specify this information, so it sounds like we may need to add a new accessibility property.

It would be best to begin the discussion of what the property should be named and what values it should accept here, as we can add properties to the `a11y:` namespace without updating the specification (epubcheck won't complain).

Once we have a solid proposal in place, we can look at whether the new property can be added to the accessibility standard, but this may qualify as a class 4 change.

**Comment by mattgarrish** [comment]
@gregoriopellegrino can you post the new ONIX values so we can see what we're working with? I don't see them in the latest release, so I assume they're still proposals.

**Comment by gregoriopellegrino** [comment]
New codes proposed for ONIX issue 62 are:

- **75 EAA exception 1 – Micro-enterprises**: Digital product falls under European Accessibility Act exception for Microenterprises (eg enterprises employing fewer than 10 people and with annual turnover or balance sheet total not exceeding €2 million). <ProductFormFeatureDescription> may carry details of the exception.
- **76 EAA exception 2 – Undue burden**: Digital product falls under European Accessibility Act exception for Undue burden (the product may not have to comply if doing so would either change the nature of the product or service offered or if it would financially overburden the publisher).
- **77 EAA exception 3 – Substantial modification**: Digital product falls under European Accessibility Act exception for Substantial modification (the product may not have to comply if doing so requires a substantial modification of the
product or service).

**Comment by mattgarrish** [comment]
The really tough question here is how specific or generic do want this metadata to be. Will it be needed elsewhere?

If we want to make the property generic, a name like `a11y:exception` might be best, otherwise we could adapt the ONIX name to make it very specific (`a11y:EAAexception`).

The values for the EAA could be exactly what's already defined: "micro-enterprises", "undue burden", and "substantial modification".

One option would be to define a generic property that we eventually add to the vocabulary in the accessibility specification (this could be below class 3 if the spec itself doesn't require the metadata), and have a separate WG/CG note that specifies the values and how to use them for the EAA.
